_maps_2024/05-aki.md

```diff
@@ -0,0 +1,9 @@
+---
+day: 5
+creator: Akira Di Sandro
+title: My Winter Vacation '22-'23
+thumbnail: /assets/thumbnails/2024/05-Aki-journey.png
+source: https://github.com/disandroa/30DayMappingChallenge2024/blob/main/scripts/Day05.R
+---
+
+I still need to add a couple details, but putting this out here for now...
```

Comment:
Hey @disandroa, I recommend you just remove the text here for now. When you don't include any content here then the default thing that the site will do is add your map image to the page. If you _do_ include content here, then you probably want to add your image manually, like [this](https://github.com/weitzman-musa/30DayMapChallenge/blob/772c6ed626df6f1b729e4aefa297f9c923b951bf/_maps_2024/04-sean.md?plain=1#L11).

```suggestion
```